utils/testcase.py

```diff
@@ -14,6 +15,7 @@ def reload(self, item):
         return item.__class__.objects.get(pk=item.pk)
 
     def setUp(self):
+        settings.TESTING = True
```

Comment:
Oh, i see the bug...
In setting, if argv[1] == 'test', then TESTING= true.  (the command is ./manage.py test'
On windows, you're using 'python manage.py test', so argv[2] == 'test'
Better fix in settigns file :D